README.md

```diff
@@ -0,0 +1,181 @@
+<a name="readme-top"></a>
```

Comment:
_note: Due to the backend migrations issue, I wasn't able to run the frontend project.  This will be taken a look in the next review._

# Frontend review

- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Itswali/Rental-Cars/assets/57421823/e81f5c44-ae05-4a32-9f64-e2226cf09a7b)


That way, your code will meet the latest standards. 💪

- [ ] This is the test that comes with React, kindly remove it (just delete the file):

![image](https://github.com/Itswali/Rental-Cars/assets/57421823/bd8a9a15-4fa2-432f-8558-198343b96648)